files/scripts/supervisor-proc-exit-listener

```diff
@@ -90,12 +90,12 @@ def generate_alerting_message(process_name, status, dead_minutes):
                   .format(process_name, status, namespace, dead_minutes))
 
 
-def get_autorestart_state(container_name):
+def get_autorestart_state(container_name, use_unix_socket_path):
     """
     @summary: Read the status of auto-restart feature from Config_DB.
     @return: Return the status of auto-restart feature.
     """
-    config_db = swsscommon.ConfigDBConnector()
+    config_db = swsscommon.ConfigDBConnector(use_unix_socket_path=use_unix_socket_path)
```

Comment:
This script need be start by supervisord.conf, according VSO and PR description: "which would fail at non-host network mode container"

So where is the supervisord.conf change for "non-host network mode container"?

Reply:
We are working on SONiC dhcp_server feature, would add a new bridge mode container, here's the supervisorcd.conf: https://github.com/sonic-net/sonic-buildimage/pull/16547/files#diff-1432e98b2e273cefb74f12d3416bcdf5dda5a27a81fea0130134bf5aaaeaf3cb